packages/yourloops/test/unit/components/patient/patient-row.test.tsx

```diff
@@ -116,21 +119,20 @@ describe('Patient row', () => {
     expect(screen.findByText('pending-invitation')).not.toBeNull()
   })
 
-  it('should display correct fields when logged in user is HCP', () => {
+  it('should display correct fields when logged in user is HCP not in remote monitoring', () => {
     render(getPatientRowJSX())
     const cells = screen.getAllByRole('cell')
-    expect(cells).toHaveLength(10)
+    expect(cells).toHaveLength(8)
     expect(within(cells[1]).queryByText('user-name')).not.toBeNull()
     expect(within(cells[2]).queryByText('N/A')).not.toBeNull()
-    expect(within(cells[3]).queryByText('no')).not.toBeNull()
-    expect(within(cells[4]).queryByText(`${patient.alarms.timeSpentAwayFromTargetRate}%`)).not.toBeNull()
-    expect(within(cells[4]).queryByTitle('time-away-alert-icon')).toBeNull()
-    expect(within(cells[5]).queryByText(`${patient.alarms.frequencyOfSevereHypoglycemiaRate}%`)).not.toBeNull()
-    expect(within(cells[4]).queryByTitle('severe-hypo-alert-icon')).toBeNull()
-    expect(within(cells[6]).queryByText(`${patient.alarms.nonDataTransmissionRate}%`)).not.toBeNull()
-    expect(within(cells[4]).queryByTitle('no-data-alert-icon')).toBeNull()
-    expect(within(cells[7]).queryByText('N/A')).not.toBeNull()
-    expect(within(cells[8]).queryByTitle('unread-messages-icon')).toBeNull()
+    expect(within(cells[3]).queryByText(`${patient.alarms.timeSpentAwayFromTargetRate}%`)).not.toBeNull()
+    expect(within(cells[3]).queryByTitle('time-away-alert-icon')).toBeNull()
+    expect(within(cells[4]).queryByText(`${patient.alarms.frequencyOfSevereHypoglycemiaRate}%`)).not.toBeNull()
+    expect(within(cells[3]).queryByTitle('severe-hypo-alert-icon')).toBeNull()
+    expect(within(cells[5]).queryByText(`${patient.alarms.nonDataTransmissionRate}%`)).not.toBeNull()
+    expect(within(cells[3]).queryByTitle('no-data-alert-icon')).toBeNull()
+    expect(within(cells[6]).queryByText('N/A')).not.toBeNull()
+    expect(within(cells[7]).queryByTitle('unread-messages-icon')).toBeNull()
```

Comment:
should we check that if user is not in remote monitoring ? cells[7] is not remove patient button ?